tests/e2e/specs/utils/confluentCloud.ts

```diff
@@ -59,29 +42,12 @@ async function handleAuthFlow(
     await authPage.locator("[name=password]").fill(password);
     await authPage.locator("[type=submit]").click();
 
-    // Wait for success page
-    try {
-      await authPage.waitForSelector("text=Authentication Complete", { timeout: 3000 });
-    } catch (error) {
-      throw new Error("Authentication failed.");
-    }
```

Comment:
I'm still not super comfortable with `expect()` calls in these utility functions, but better to use the auto-await than `.waitForSelector()`?